src/bertron_schema/schema/bertron_schema.yaml

```diff
@@ -1,96 +1,288 @@
+  Entity:
     class_uri: schema:Thing
+    description: |
+      An object retrieved by BERtron from a BER data API.
+    attributes:
+      ber_data_source:
+        description: The BER member from whence the entity originated.
+        range: BERSourceType
+        required: true
+      # TODO!
+      credit_metadata:
+        description: Citation metadata for the entity.
+        todos:
+        - add in KBase credit metadata schema
+      coordinates:
+        description: The geographic coordinates associated with an entity. For entities with a bounding box, the centroid is used as the geographic reference.
+        range: Coordinates
+        required: true
+      data_type:
+        description: What kind of data this entity is -- e.g. sequence data; a soil core; a well; etc.
+        range: DataTypeTagType
+        multivalued: true
+        required: true
+      description:
+        description: Textual description of the entity.
+        required: false
+        slot_uri: schema:description
+        examples:
+        - value: "River water sample taken by AquaTROLL 9000."
+        - value: "Genome sequence of P. aeruginosa strain IDDQD"
+      id:
+        aliases:
+        - BER data source internal identifier
+        - CURIE
+        description: The unique ID used for the entity within the BER resource. It may not necessarily be resolvable outside the resource.
+        comments:
+        - If the data source does not use CURIEs, we cannot guarantee that IDs will be unique between all the BER sources.
+        # identifier: true
+        # required: true
+        range: string
+        slot_uri: schema:identifier
+      name:
+        description: Human-readable string representing an entity.
+        multivalued: false
+        slot_uri: schema:name
+        examples:
+        - value: "Pseudomonas aeruginosa strain IDDQD"
+        - value: "Soil core FW-106"
+      alt_ids:
+        aliases:
+        - CURIEs
+        - database cross-references
+        - dbxrefs
+        - IDs
+        - alternative identifiers
+        - alternative IDs
+        - alternative PIDs
+        - PIDs
+        description: Fully-qualified URI or CURIE used as an identifier for an entity.
+        range: uriorcurie
+        multivalued: true
+        examples:
+          - value: "NCBItaxon:172684329"
+          - value: "ISGN:1986497"
+        comments:
+        - The entity `id` should not appear in this list.
+      alt_names:
+        aliases:
+        - alternative names
+        - synonyms
+        description: Textual identifiers for an entity.
+        range: Name
+        multivalued: true
+        comments:
+        - The entity `name` should not appear in this list.
+      part_of_collection:
+        description: Administrative collection (e.g. project, campaign, whatever) that the entity was generated as part of. May also be called a project.
+        multivalued: true
+        required: false
+        range: DataCollection
```

Comment:
This is an attempt to handle redundancy in citation data where numerous entities have the same credit metadata (CM). I'm not sure if this is actually necessary or whether we could instead capture this info in the CM and have the `credit_metadata` field be a reference to a unique ID for the CM.

e.g. search results format:

```js
"results": [
  {"id": "db:id", "credit_metadata_id": "db:proj_1234", ...},
  {"id": "db:id_2", "credit_metadata_id": "db:proj_1234", ...},
  {"id": "db:id_3", "credit_metadata_id": "db:proj_5678", ...},
],
"credit_metadata": {
"db:proj_1234": { ...credit metadata here...},
"db:proj_5678": { ...credit metadata here...},
}
```